### DISCUSSION TITLE
Considering weighted graphs in DynBetweennessOneNode algorithm.

### DISCUSSION THREAD

**Discussion by davisv7** [networkit/networkit#990] [Ideas]
https://github.com/networkit/networkit/blob/879379178a809d40d5840cbfc7825a36069cd943/networkit/centrality.pyx#L547

Can this algorithm be extended to work for weighted graphs as well?

**Comment by fabratu** [comment]
Hi,

thanks for the suggestion. We are currently working on a integration of several dynamic betweenness algorithms (see #624). Though OneNode for weighted graphs is not among them, this can be added to the list of changes. It currently is already aware of weighted graphs, implementation for this has to be finished.
